DominicGBauer/information-access
# feat: added preview

# Pull Request

## Scope

Added in a preview of the pdf via the results card. 
https://dev.azure.com/JHNMAT015/ECO5037S%20Front%20end/_backlogs/backlog/ECO5037S%20Front%20end%20Team/Backlog%20items/?workitem=130

## Work Done
1. Added a new view that shows a pdf of the paper selected.
2. Added a magnifying glass icon on the results cards that links to the preview. 

## Steps to Test
1. ```npm run start```
2.  Do a search that provides results. 
3. Click on the magnifying glass icon.
4. See the preview. 

## Screenshot

![image](https://user-images.githubusercontent.com/80746660/135803360-0b618578-0730-4808-8e32-5bf3d3a28dd8.png)

![image](https://user-images.githubusercontent.com/80746660/135803427-9cbfcb65-235d-49e9-8111-723d501f1116.png)
